.github/workflows/pre-commit.yml

```diff
@@ -23,3 +23,25 @@ jobs:
       - uses: pre-commit/action@v3.0.0
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+
+  check-chart-version-bump:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4.0.0
+
+      - name: Check if Chart.yaml versions are bumped if relevant
+        run: |
+          CHARTS="endpoint events logs metrics proxy stack traces"
```

Comment:
I can't imagine there's a reason to not check every single chart, in which case a `find` here looking for Chart.yaml might be more appropriate.